core/domain/domain.core.yml

```diff
@@ -21,6 +21,9 @@ intents:
   - no_change
   - daily_checkin:
       use_entities: []
+  - send_validation_code:
+      triggers: action_send_validation_code
+
 entities:
   - province
   - metadata
```

Comment:
first_name and validation_code entities are not there and it doesn't trigger an error to send a playload with them? This is weird... Anyway it would be cleaner to have them in the domain.

Reply:
Definitely!